Source/SourceKittenFramework/String+SourceKitten.swift

```diff
@@ -101,14 +120,22 @@ extension NSString {
         }
 
         var lines: [Line] {
-            return rangeLinePairs.map { $0.line }
+            return rangesAndLine.map { $0.line }
         }
 
         func lineAndCharacterForCharacterOffset(offset: Int) -> (line: Int, character: Int)? {
-            let index = rangeLinePairs.indexOf { NSLocationInRange(offset, $0.range) }
+            let index = rangesAndLine.indexOf { NSLocationInRange(offset, $0.range) }
             return index.map {
-                let pair = rangeLinePairs[$0]
-                return (line: pair.line.index, character: offset - pair.range.location + 1)
+                let element = rangesAndLine[$0]
+                return (line: element.line.index, character: offset - element.range.location + 1)
+            }
+        }
+        
+        func lineAndCharacterForByteOffset(offset: Int) -> (line: Int, character: Int)? {
+            let index = rangesAndLine.indexOf { NSLocationInRange(offset, $0.byteRange) }
+            return index.map {
+                let element = rangesAndLine[$0]
+                return (line: element.line.index, character: offset - element.range.location + 1)
```

Comment:
@jpsim: Sorry, I have a bug here. The `character` is not correct. I'll fix that.